libvalkeycluster/valkeycluster.c

```diff
@@ -197,8 +197,8 @@ static unsigned int keyHashSlot(char *key, int keylen) {
     return crc16(key + s + 1, e - s - 1) & 0x3FFF;
 }
 
-static void __redisClusterSetError(redisClusterContext *cc, int type,
-                                   const char *str) {
+static void __valkeyClusterSetError(valkeyClusterContext *cc, int type,
```

Comment:
branch `rename` removed the double underscore as function name prefix in commit "Fix ISO C naming violations"

Reply:
Separate PR

Reply:
Covered by issue #19 and PR #21